selfservice/strategy/webauthn/login_test.go

```diff
@@ -414,6 +414,7 @@ func TestCompleteLogin(t *testing.T) {
 
 		t.Run("case=succeeds with passwordless login", func(t *testing.T) {
 			run := func(t *testing.T, spa bool) {
+				conf.MustSet(ctx, config.ViperKeySessionWhoAmIAAL, "aal1")
```

Comment:
@aeneasr Please take a look at why this test passes in master without this configured.